src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -72,6 +73,8 @@ protected Experience[] get(final Event e) {
 			return new Experience[] {new Experience(((ExperienceSpawnEvent) e).getSpawnedXP())};
 		else if (e instanceof BlockBreakEvent)
```

Comment:
Unable to comment at L71, refractor `e` to `event`

Reply:
I think this is outside the scope of this PR. See [comment](https://github.com/SkriptLang/Skript/pull/5389#discussion_r1084344129)